examples/tv-app/android/java/TVApp-JNI.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ *   Copyright (c) 2021 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+/**
+ *    @file
+ *      Implementation of JNI bridge for Tv App on Android.
```

Comment:
nit: this `@file` comment does not seem to help me understand code more: it is obvious that a file named  `TVApp-JNI.cpp` contains an implementation of JNI for a TV APP

Reply:
OK, I will remove it in next PR